core/src/main/java/org/apache/iceberg/BaseMetastoreTableOperations.java

```diff
@@ -88,11 +88,9 @@ public TableMetadata refresh() {
       if (currentMetadataWasAvailable) {
         LOG.warn("Could not find the table during refresh, setting current metadata to null", e);
       }
-
```

Comment:
Can you add this line back? Removing it will just cause commit conflicts and it isn't needed for the PR.